node-tests/helpers/EmberCheckupFixturifyProject.ts

```diff
@@ -0,0 +1,11 @@
+const FixturifyProject = require('ember-cli/tests/helpers/fixturify-project');
+const rimraf = require('rimraf');
+
+module.exports = class EmberCheckupFixturifyProject extends FixturifyProject {
+  dispose(filepath?: string) {
```

Comment:
Should `filePath` be renamed to `tempFilesToCleanupPath`? (or something like that)